pallets/omnipool/src/lib.rs

```diff
@@ -423,6 +423,10 @@ pub mod pallet {
 		ZeroAmountOut,
 		/// Existential deposit of asset is not available.
 		ExistentialDepositNotAvailable,
+		/// Liquidity limit is not reached while removing liquidity
+		LiquidityLimitNotReached,
+		/// Shares limit is not reached while adding liquidity
+		SharesLimitNotReached,
```

Comment:
I would just create one error called `SlippageLimit`. 

i believe there is no real need to have more detailed error message.  we have similar thing in stableswap, dca has something similar too. it would be good to be consistent accross amms and similar things.